.circleci/config.yml.j2

```diff
@@ -171,30 +175,63 @@ workflows:
+      - download-file:
+          name: download-latest-{{ airflow_version | replace('.dev', '') }}-build-info
+          url: https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build.json
+          file: latest-{{ airflow_version | replace('.dev', '') }}.build.json
+      - write-version-to-tag-file:
+          name: write-version-to-tag-file-{{ airflow_version | replace('.dev', '') }}
+          input_json_file: latest-{{ airflow_version | replace('.dev', '') }}.build.json
+          output_file: extra-tags-{{ airflow_version | replace('.dev', '') }}.txt
+          requires:
+            - download-latest-{{ airflow_version | replace('.dev', '') }}-build-info
+      {%- endif %}{# edge_build #}
+
+      {%- for distribution in distributions %}
       - build:
           name: build-{{ airflow_version }}-{{ distribution }}
           airflow_version: {{ airflow_version }}
           distribution_name: {{ distribution }}
           dev_build: true
-          extra_args: "--build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build)"
+          extra_args:
+            --build-arg VERSION=$(jq -r '.output.airflow.package.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            {%- if edge_build %}
+            --label org.apache.airflow.ci.build.date=$(jq -r '.date' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.ci.build.url=$(jq -r '.github.run.url' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.ci.build.version=$(jq -r '.output.airflow.package.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.ci.js.node.version_string=$(jq -r '.js.node.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.ci.js.npm.version_string=$(jq -r '.js.npm.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.ci.js.yarn.version_string=$(jq -r '.js.yarn.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.ci.python.version_string=$(jq -r '.python.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.git.commit_sha=$(jq -r '.git.commit' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label org.apache.airflow.git.$(jq -r '.git.ref.type' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)=$(jq -r '.git.ref.name' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
+            --label io.astronomer.astronomer_certified.build.version=$(jq -r '.output.astronomer_certified.package.version' <latest-{{ airflow_version | replace('.dev', '') }}.build.json)
```

Comment:
Let's add a test in https://github.com/astronomer/ap-airflow/blob/master/.circleci/bin/test-airflow-image.py to check these labels exists and are _not empty_ for edge builds

https://github.com/astronomer/ap-airflow/blob/5572d30145a79f29cff320ee35b879eadda52f8e/.circleci/bin/test-airflow-image.py#L30

Reply:
Fixed.